tp

```diff
@@ -0,0 +1,14 @@
+package.AWSSearchServicesKibanaSecurityAnalyticsPlugin = {
```

Comment:
Is this file addition unintentional?

why is it 2.11?

Reply:
Not sure how this file ended up here. Removed it.